### PR TITLE
[BaseKit.com] Remove broken targets

### PR DESCRIPTION
**Pull request by Bisaloo**
I'm not entirely sure what to do here. I usually wouldn't use https-everywhere to fix broken websites but it was already here so...

### CODE REVIEW

File: src/chrome/content/rules/basekit.com.xml [hunk 1]
```diff
@@ -1,79 +1,32 @@
...
-
-	Insecure cookies are set for these hosts: ᶜ
-
-		- odense.basekit.com
-
-	ᶜ See https://owasp.org/index.php/SecureFlag
-
+	Invalid certificate:
+		basekit.com
+		custom-css.basekit.com
```
Comment (J0WI) [inline src/chrome/content/rules/basekit.com.xml]: This is an active target

File: src/chrome/content/rules/basekit.com.xml [hunk 2]
```diff
@@ -1,79 +1,32 @@
...
 	<target host="resellers.basekit.com" />
 	<target host="scottishculture.basekit.com" />
-	<target host="www.basekit.com" />
-
-	<!--	Complications:
-				-->
-	<target host="custom-css.basekit.com" />
-	<target host="resources.basekit.com" />
-
+	<target host="try.basekit.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/basekit.com.xml]: See also:
```
support.basekit.com
apidocs.basekit.com
knowledge.basekit.com
docs.basekit.com
partners.basekit.com
trivia.basekit.com
```

It seems that customers get a subdomain (which works over HTTPS), so this rule may need a wildcard.

File: src/chrome/content/rules/basekit.com.xml (hunk 1 above)
Comment (Bisaloo) [inline src/chrome/content/rules/basekit.com.xml]: Yes, see the rewrite rule.

File: src/chrome/content/rules/basekit.com.xml (hunk 2 above)
Comment (Bisaloo) [inline src/chrome/content/rules/basekit.com.xml]: > It seems that customers get a subdomain (which works over HTTPS), so this rule may need a wildcard.

Some of them may have mixed content issues (for example https://odense.basekit.com/). Should I still add a wildcard with exclusion and hope it doesn't break too many customers subdomains?

Comment (J0WI) [inline src/chrome/content/rules/basekit.com.xml]: As far as I understand this host is a website builder. It's unlikely that there are major differences between all the sites or customers are allowed to insert active content. In this example only Google Fonts are affected.
Let's add the wildcard.

**Pull request merged**